### DISCUSSION TITLE
Differentiate inventory from different sites with same name

### DISCUSSION THREAD

**Discussion by mattpltn** [bb-Ricardo/netbox-sync#255] [General]
Hello,

We have two sites that have the same datacenter and cluster name. I know this is very stupid but the team does not want to change it because it is in production.

So in the inventory the VMs from both sites gets populate in the same cluster.

I was wondering if there is any way on the script to alter the datacenter name or to force the cluster name so I can populate in a different cluster?

**Comment by bb-Ricardo** [comment]
Hi,

This should be actually working but there is also a open issue about this topic.

I haven't had time lately to look into it but that is definitely to be fixed.

Sorry for the inconvenience.

**Comment by bb-Ricardo** [comment]
Hi, same datacenter AND same cluster name. There is no way I know of how the script should keep these two datacenter separated. The only distinction between ClusterGroups in NetBox is the Name.
